package.json

```diff
@@ -8,7 +8,8 @@
   },
   "scripts": {
     "start": "node ./server/bin.js",
-    "test": "grunt test"
+    "test": "grunt test",
+    "postinstall": "./node_modules/bower/bin/bower install"
```

Comment:
The thing is that I wouldn't want everyone installing it via NPM to go through the extra build/installation step.